exampleCourse/questions/templateFixedMultipleChoiceManyAnswers/question.html

```diff
@@ -0,0 +1,20 @@
+<pl-question-panel>
+What color can be found in a rainbow?
+</pl-question-panel>
+
+<pl-multiple-choice answers-name='colors' number-answers='4'>
+  <pl-answer correct='true'>Red</pl-answer>
+  <pl-answer correct='true'>Blue</pl-answer>
+  <pl-answer correct='true'>Purple</pl-answer>
```

Comment:
```suggestion
  <pl-answer correct='true'>Violet</pl-answer>
```

Is violet/purple a UK/US thing? But see https://en.wikipedia.org/wiki/Rainbow#Number_of_colours_in_spectrum_or_rainbow

Reply:
Seem so, most people here say "purple" instead of "violet" but still learn the spectrum of the rainbow as 'ROYGBIV' -- with violet at the end.